ee/modules/350-node-local-dns/templates/daemonset.yaml

```diff
@@ -150,7 +150,6 @@ spec:
         image: {{ include "helm_lib_module_common_image" (list . "kubeRbacProxy") }}
         args:
         - "--secure-listen-address=$(KUBE_RBAC_PROXY_LISTEN_ADDRESS):9254"
-        - "--client-ca-file=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
```

Comment:
Why don't you mount and use the new one?

Reply:
I added this mount only where it is required.